ice-cream-factory-datapoints-extractor/pyproject.toml

```diff
@@ -1,5 +1,5 @@
 [tool.poetry]
-name = "ice cream factory datapoints extractor"
+name = "rest_extractor"
```

Comment:
why rename to rest_extractor? I would find that a bit confusing especially as it is not using the rest extension.

I think `ice_cream_extractor` or something similar is a better name, and entrypoint 🙂

Reply:
done